src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.tsx

```diff
@@ -131,6 +138,10 @@ function Panel({
     };
   }, [editor]);
 
+  useEffect(() => {
```

Comment:
:hand: This doesn't need to be in an effect - we should be able to just calculate it directly while rendering based on the known variables.

Reply:
Actually, reading DOM layout (global mutable value) isn't really safe to do during render because it will produce unpredictable results for the given input (props) based on whatever the global mutable state happens to be.

Maybe we could do this using `useSyncExternalStore` though?